CounterApp/Lock/LockCore.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  LockCore.swift
+//  CounterApp
+//
+//  Created by Wenjuan Li on 2022/7/12.
+//
+
+import ComposableArchitecture
+import Foundation
+
+struct LockState: Equatable {
+    var counters: IdentifiedArrayOf<CounterState> = []
+    var isPresent: Bool = false
+    var unlockAlert: AlertState<LockAction>?
+}
+
+enum LockAction: Equatable {
+    case counter(id: CounterState.ID, action: CounterAction)
+    case setSheet(isPresented: Bool)
```

Comment:
Same here. This action should be part of `AppAction` enum.

Reply:
@nanocba Thanks! Fixed.